components/LinkCustom/LinkCustom.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import LinkCustom from './LinkCustom';
+
+describe('LinkCustom Component', () => {
+  it('renders with default props and does not open in a new tab', () => {
+    render(<LinkCustom href="https://example.com">Example Link</LinkCustom>);
+    const link = screen.getByTestId('link-custom');
+    expect(link).toBeInTheDocument();
+    expect(link).not.toHaveAttribute('target', '_blank');
+  });
+
+  it('opens in a new tab when newTab is true', () => {
+    render(
+      <LinkCustom href="https://example.com" newTab>
+        Example New Tab Link
+      </LinkCustom>,
+    );
+    const link = screen.getByTestId('link-custom');
+    expect(link).toBeInTheDocument();
+    expect(link).toHaveAttribute('target', '_blank');
+    expect(link).toHaveAttribute('rel', 'noopener noreferrer');
+  });
+});
```

Comment:
Should the unit test also check the default and hover styles?

Reply:
I suppose you are right! Nice catch; I'll add it in my next update.

Reply:
The more I think about this, checking for the appropriate classes should be more than enough. Instead of writing another test for this, I'll just do that.

Reply:
So right now the default color of the link is not different from the rest of the sentence, right? That's why you are only checking the hover classes.

Reply:
> So right now the default color of the link is not different from the rest of the sentence, right? That's why you are only checking the hover classes.

That is correct. However, I believe that the link should be underlined to differentiate it from the rest of the sentence. Would some sort of aria-label also make sense @CorinaMurg?

Reply:
This has been added in the latest commit